### PR TITLE
Add latestTasks property to Worker

### PR DESCRIPTION
**Pull request by helfi92**

### CODE REVIEW

**Comment by helfi92** [comment]
@eliperelman I am not able to flag you for review here. Could you give me more access, please? Thanks.

File: src/resolvers/Workers.js [hunk 1]
```diff
@@ -1,4 +1,12 @@
 export default {
+  Worker: {
+    // eslint-disable-next-line no-empty-pattern
```
Comment (eliperelman) [inline src/resolvers/Workers.js]: Instead of this override, just make the middle argument `args`.

File: src/graphql/Workers.graphql [hunk 2]
```diff
@@ -13,16 +13,40 @@ type WorkerCompact {
...
+  # Note that a quarantine can be lifted by setting quarantineUntil to the present time (or somewhere in the past).
   quarantineUntil: DateTime
+
+  # Date of the first time this worker claimed a task.
   firstClaim: Date!
+
+  # A list of actions exposed at the worker level.
   actions: [ProvisionerAction]!
+
+  # A list of most recent tasks claimed by the worker
```
Comment (eliperelman) [inline src/graphql/Workers.graphql]: Nit: period at end of sentence.

File: src/graphql/Workers.graphql [hunk 3]
```diff
@@ -13,16 +13,40 @@ type WorkerCompact {
   workerType: String!
 }
 
+# Definition of a worker
 type Worker {
+  # Provisioner ID name.
```
Comment (eliperelman) [inline src/graphql/Workers.graphql]: This should probably just be "Provisioner identifier", no name.

File: src/graphql/Workers.graphql [hunk 4]
```diff
@@ -13,16 +13,40 @@ type WorkerCompact {
...
   provisionerId: String!
+
+  # Worker type name.
   workerType: String!
+
+  # Identifier for group that worker who executes this run is a part of,
```
Comment (eliperelman) [inline src/graphql/Workers.graphql]: Nit: end of sentence, end with period. Make next line a sentence with capitalized letter.

**Pull request merged**